lib/scrabble.rb

```diff
@@ -1,18 +1,51 @@
 class Scrabble
 
+  def initialize
+    @array_of_word = []
+    @count = 0
+  end
+
   def score(word)
-    1
+    if word.length > 6
+      @count += 10
+    end
+    @array_of_word =[]
+    word.split('').each do |letter|
+      @array_of_word << letter.upcase
+    end
+      point_counter(@array_of_word)
```

Comment:
Since @array_of_word is an instance variable you don't need to pass it to point_counter. The method point_counter will have access to any instance variables.